datareduction/datareducer.cpp

```diff
@@ -2962,7 +2962,21 @@ void initializeDataReducers(DataReducer * outputReducer, DataReducer * diagnosti
       }
       if(P::systemWriteAllDROs || lowercase == "vg_amr_alpha2") {
          outputReducer->addOperator(new DRO::DataReductionOperatorCellParams("vg_amr_alpha2",CellParams::AMR_ALPHA2,1));
-         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\alpha_2","");
+         outputReducer->addMetadata(outputReducer->size()-1,"","","$\\alpha_2$","");
```

Comment:
Can you add DROs for the velocity derivatives? We have a flag for all derivatives, and allow them individually too iirc.

Reply:
OK that'll come in in an upcoming PR.